pkg/morph/client/static.go

```diff
@@ -34,7 +34,7 @@ type staticOpts struct {
 //
 // See also TryNotary.
 func (s *StaticClient) WithNotary() bool {
-	return s.tryNotary
+	return s.client.IsNotaryEnabled()
```

Comment:
What about `TryNotary` option?

Reply:
about trying notary invokes (not all the calls should be notary even in notary-enabled mode)

`notaryEnabled && tryNotary` => invoke notary call
`notaryDisabled && !tryNotary` => regular call